### ISSUE TITLE
2D Editor should not allow moving controls (and show a warning) when editing a child of a container.

### ISSUE THREAD

**Issue by reduz** [godotengine/godot#20750]
Users are often confused by this, trying to move children of containers and then when the game runs or they re-open the editor, lose their changes.

**Comment by eon-s** [comment]
"now "or "not on the title?

**Comment by bojidar-bg** [comment]
@Zireael07 I still haven't seen a Container which does not enforce some sort of layout, can you give an example? (and why wouldn't a Control or ReferenceFrame do the job as well)

**Comment by groud** [comment]
I'm waiting on #20587 to be merged before working on that

**Comment by Zireael07** [comment]
I meant this node (bordered in red). As far as I can tell, it doesn't enforce any layout and I can arrange/move the child controls any way I want :) (The scene in the screen gets several more child nodes when run and the children of the container can freely overlap and whatever)

![godot_v3 0 3-stable_win64_2018-08-06_08-46-14](https://user-images.githubusercontent.com/4525388/43701090-4461b878-9955-11e8-8c3e-426f325216a2.png)

**Comment by bojidar-bg** [comment]
@Zireael07 You should use a plain Control node for that.

**Comment by akien-mga** [comment]
> I'm waiting on #20587 to be merged before working on that

Now's your time :)

**Comment by eon-s** [comment]
An observation, resizing Controls inside a container, should affect min_size, not size, size is useless.

**Comment by bojidar-bg** [comment]
That would be good, but it should be disabled for "Expand" directions, since it might make the container unable to shrink.

**Comment by groud** [comment]
Closing as superseded by #20750
